### DISCUSSION TITLE
Signed NuGet package

### DISCUSSION THREAD

**Discussion by beggarstudios** [candoumbe/DataFilters#331] [Q&A]
Hi, I was wondering if there was a signed version of the NuGet package so we can implement DataFilters in a solution that requires Strong Naming.

**Comment by candoumbe** [comment]
Hello !
Sorry for not answering sooner to your request but I did not get your request.
I will include that in my pipeline.

May I ask how did you come across this library ?

Do not hesitate to ⭐  the repository
